
public void run() throws IOException {
    
    int TEST_CASE = 1;
    while (TEST_CASE-- != 0) {
        cs = io.next();
        io.out.println(rec(0, 0, 0));
    }
}